service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -90,9 +90,10 @@ sudo mv nextflow /usr/bin/nextflow
 
 # Install cromwell
 readonly CROMWELL_LATEST_VERSION="81"
+readonly CROMWELL_JAR_PATH="/usr/shared/java/"
```

Comment:
oh i didn't mean having env variable in the script. that's OK too but I mean setting this in the bash.profile so the environment get set everytime a new processer started (e.g. a user opens a terminal). this is so that the user doesn't need to remember the fall path of cromwell jar. 

echo "export CROMWELL_JAR='/usr/shared/java/cromwell-${CROMWELL_LATEST_VERSION}".jar' >> "/home/${JUPYTER_USER}/.bash_profile"

Reply:
Oh. Sorry for the misunderstanding. Yesterday, I also checked the ticket comment (my note).  I saw the echo sentence, but feel really strange to add path into .jar file. because jar file does not readable. thanks for making it clear, my note is wrong....
I will keep the path env variable here to make consistence and easy to modify. Thanks again for the chance to learn the script.